qir/qat/ValidationPass/FunctionValidationPass.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Logging/ILogger.hpp"
+#include "Profile/Profile.hpp"
+#include "QatTypes/QatTypes.hpp"
+#include "StaticResourceComponent/AllocationAnalysisPass.hpp"
+#include "ValidationPass/FunctionValidationPass.hpp"
+
+#include "Llvm/Llvm.hpp"
+
+#include <functional>
+#include <unordered_map>
+#include <vector>
+
+namespace microsoft
+{
+namespace quantum
+{
```

Comment:
Doesn't the QAT compiler and chosen C++ standard support this?
```suggestion
namespace microsoft::quantum
{
```

Reply:
I wasn't aware that C++14 has support for this. It is a good suggestion and I would propose to make a separate PR where this is changed repository wide and not just in a single file.

Reply:
I'm leaving this decision to you 
whether to make it in a separate PR repo-wide 
or in the new files and then at some time in a separate PR repo-wide
(or other or not do :-) ).